File: Habitica/src/main/java/com/habitrpg/android/habitica/data/local/CustomizationLocalRepository.java
package com.habitrpg.android.habitica.data.local;

import com.habitrpg.android.habitica.models.inventory.Customization;

import io.realm.RealmResults;
import rx.Observable;

public interface CustomizationLocalRepository extends ContentLocalRepository {
    Observable<RealmResults<Customization>> getCustomizations(String type, String category, boolean onlyAvailable);
}

File: Habitica/src/main/java/com/habitrpg/android/habitica/data/local/CustomizationLocalRepository.kt
package com.habitrpg.android.habitica.data.local

import com.habitrpg.android.habitica.models.inventory.Customization
import kotlinx.coroutines.flow.Flow

interface CustomizationLocalRepository : ContentLocalRepository {
    fun getCustomizations(
        type: String,
        category: String?,
        onlyAvailable: Boolean
    ): Flow<List<Customization>>
}
